core/nwb.image.yaml

```diff
@@ -105,16 +102,101 @@ groups:
   - name: format
     dtype: text
     default_value: raw
-    doc: Format of image. If this is 'external', then the attribute 'external_file'
-      contains the path information to the image files. If this is 'raw', then the raw
-      (single-channel) binary data is stored in the 'data' dataset. If this attribute
+    doc: Format of image. If this is 'raw', then the raw (single-channel) binary data is 
+      stored in the 'data' dataset. If this is 'external', then the attribute 'external_file'
+      contains the path information to the image files. Use of external files is discouraged
+      and will be deprecated. Please use ExternalImageSeries instead. If this attribute
       is not present, then the default format='raw' case is assumed.
     quantity: '?'
   links:
   - name: device
     target_type: Device
     doc: Link to the Device object that was used to capture these images.
     quantity: '?'
+    
+- neurodata_type_def: ExternalImageSeries
+  neurodata_type_inc: NWBDataInterface
+  doc: General image data stored in external files on the filesystem. Note that this type
+    does not extend TimeSeries because there is no 'data' dataset. However, there is still
+    a time dimension which can be stored as timestamps or a starting time and sampling rate. 
+  datasets:
+  - name: dimension
+    dtype: int32
+    dims:
+    - rank
+    shape:
+    - null
+    doc: Number of pixels on x, y, (and z) axes.
+    quantity: '?'
```

Comment:
Is this necessary? It also assumes that all images are of the same size.

Reply:
I think this makes sense to keep.